### PR TITLE
#MoreDice2016

### PR DESCRIPTION
**Pull request by Anewbe**
- Adds more dice (d4, d8, d10, d12, and d100)
- Allows you to spawn in with a full set of 7 dice

Sprites courtesy of /tg/station

### CODE REVIEW

File: code/game/objects/items/weapons/storage/misc.dm [hunk 1]
```diff
@@ -1,11 +1,25 @@
-/obj/item/weapon/storage/pill_bottle/dice
+/obj/item/weapon/storage/pill_bottle/dice	//7d6
 	name = "pack of dice"
 	desc = "It's a small container with dice inside."
 
 	New()
```
Comment (Datraen) [inline code/game/objects/items/weapons/storage/misc.dm]: Absolute pathing for procs.

File: code/game/objects/items/weapons/storage/misc.dm [hunk 2]
```diff
@@ -1,11 +1,25 @@
...
 		..()
+		for(var/i = 1 to 7)
 		new /obj/item/weapon/dice( src )
+
+/obj/item/weapon/storage/pill_bottle/dice/nerd	//DnD dice
+	name = "pack of dice"
+	desc = "It's a small container with gaming dice inside."
+
+	New()
```
Comment (Datraen) [inline code/game/objects/items/weapons/storage/misc.dm]: Absolute pathing for procs.

File: code/game/objects/items/weapons/storage/misc.dm (hunk 1 above)
Comment (Datraen) [inline code/game/objects/items/weapons/storage/misc.dm]: Absolute pathing for procs.

**Comment by Anewbe** [comment]
Oddly enough that pathing stuff is the old code, I'll fix it in the next commit.

**Comment by Anewbe** [comment]
Should be merge-able now.

**Pull request merged**